### PR TITLE
[IMP] event_create_track: Location, speaker and assistant in event tr…

### PR DESCRIPTION
**Pull request by Berezi**
…ack.

### CODE REVIEW

**Comment by Berezi** [comment]
@oihane @alfredoavanzosc @Daniel-CA @anajuaristi review please.

File: event_create_track/models/event_event.py [hunk 1]
```diff
@@ -53,5 +53,9 @@ def catch_values_for_track(self, line, date_start, n):
             'date': my_date,
             'duration': line.hour_to-line.hour_from,
             'event_id': self.id,
-            'user_id': self.env.user.id}
+            'user_id': self.env.user.id,
+            'address_id': self.address_id.id,
+            'partner_id': self.main_responsible_id.partner_id.id,
+            'second_responsible_id': self.second_responsible_id.partner_id.id}
+        print ('*** track_vals: ' + str(track_vals))
```
Comment (oihane) [inline event_create_track/models/event_event.py]: esto sobra

**Comment by Berezi** [comment]
@oihane @anajuaristi @alfredoavanzosc @Daniel-CA review please.

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

File: event_create_track/models/event_event.py (hunk 1 above)
Comment (Berezi) [reply event_create_track/models/event_event.py]: Se me ha colau. Ya lo he quitado. Barkatu!

**Review by Daniel-CA — APPROVED** [review]

**Pull request merged**